app/models/air_now/stream.rb

```diff
@@ -1,16 +1,42 @@
 module AirNow
+  pm25_threshold_set = ThresholdSet.find_or_create_by!(
+    threshold_very_low: 0,
+    threshold_low: 12,
+    threshold_medium: 35,
+    threshold_high: 55,
+    threshold_very_high: 150,
+    sensor_name: 'Government-PM2.5',
+    unit_symbol: 'µg/m³',
+  )
+
+  no2_threshold_set = ThresholdSet.find_or_create_by!(
+    threshold_very_low: 0,
+    threshold_low: 53,
+    threshold_medium: 100,
+    threshold_high: 360,
+    threshold_very_high: 649,
+    sensor_name: 'Government-NO2',
+    unit_symbol: 'ppb',
+  )
+
+  o3_threshold_set = ThresholdSet.find_or_create_by!(
+    threshold_very_low: 0,
+    threshold_low: 59,
+    threshold_medium: 75,
+    threshold_high: 95,
+    threshold_very_high: 115,
+    sensor_name: 'Government-Ozone',
+    unit_symbol: 'ppb',
+  )
+
```

Comment:
I'd like to avoid depending on the manual creation of objects on new servers as much as possible. Right now it's the case with AirNow user as well so I'll change it in the next commit.